src/pyhf/infer/calculators.py

```diff
@@ -59,6 +59,20 @@ def __init__(self, shift):
         self.shift = shift
         self.sqrtqmuA_v = None
 
+    def cdf(self, value):
+        """
+        Compute the value of the cumulative distribution function for a given value of the test statistic.
+
+        Args:
+            value (`float`): The test statistic value.
+
+        Returns:
+            probability (`float`): The integrated probability to observe a test statistic less than or equal to the observed ``value``.
```

Comment:
```suggestion
            Float: The integrated probability to observe a test statistic less than or equal to the observed ``value``.
```

Shouldn't this be the return type? This is how it is for all the rest of our docs. e.g. https://scikit-hep.org/pyhf/_generated/pyhf.tensor.pytorch_backend.pytorch_backend.html#pyhf.tensor.pytorch_backend.pytorch_backend.astensor